### PR TITLE
#5 - October release notes

### LINKED ISSUE #7: October release notes
We need to publish October release notes article to give overview of changes made in Artipie during the month

### LINKED ISSUE #724: Proxy information in examples
Let's add information about files proxy adapter to the corresponding example.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #7 
I've added release notes for October. This time I've decided to provide a list of supported proxy, but before publishing we should fix https://github.com/artipie/artipie/issues/724.
Please, let me know if I've missed any significant features.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@olegmoz could you please review? I cannot request reviews in this repo

File: _posts/2020/10/2020-10-28-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  - artipie
+  - binary artifacts
+  - hosting of artifacts
+  - maven
+  - artipie rest api
+  - users
+  - permissions
+---
+
+v0.12 was released on NN. In October, we focused on extending and adding proxy for our adapters.
```
Comment (olegmoz) [inline _posts/2020/10/2020-10-28-release-notes.md]: @olenagerasimova I do not think that we should be using word `adapter` in release notes. `adapter` is word like module, it is for people aware of Artipie's internals. Release notes are targeted at users and for them term `repository` or `repository type` would be more familiar. So I'd suggest to change wording here and in other parts of release notes

File: _posts/2020/10/2020-10-28-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+- [docker-proxy](https://github.com/artipie/artipie/tree/master/examples/docker#docker-proxy-repo) 
+(also supports `push` operation to the local storage)
+- [maven-proxy](https://github.com/artipie/artipie/tree/master/examples/maven) (multiple remotes are supported)
+- [npm-proxy](https://github.com/artipie/artipie/tree/master/examples/npm#npm-proxy-repo)
+- [pypi-proxy](https://github.com/artipie/artipie/tree/master/examples/pypi#python-proxy)
+- [files-proxy](https://github.com/artipie/artipie/tree/master/examples/binary#proxy-binary-repo)
+
+We also worked on another features in Artipie:
+- pypi-adapter now [supports](https://github.com/artipie/pypi-adapter/issues/135) `pip search` command
```
Comment (olegmoz) [inline _posts/2020/10/2020-10-28-release-notes.md]: @olenagerasimova I think it's fair to say that Artipie is now compatible with `pip search` command

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see two comments

File: _posts/2020/10/2020-10-28-release-notes.md (hunk 1 above)
Comment (olenagerasimova) [reply _posts/2020/10/2020-10-28-release-notes.md]: @olegmoz hm, I do not think "repository" is a suitable word here as "repository" is just a specifically organized set of files, and I'm talking about a library (or a module) here. Also, we've already used word "adapter" in the previous release notes, so I think it's ok to use it farther.

Comment (olegmoz) [reply _posts/2020/10/2020-10-28-release-notes.md]: @olenagerasimova ok, I see. It's okey if we are using word `adapter` already

**Comment by olegmoz** [comment]
@olenagerasimova I have started release `0.12` for Artipie here: https://github.com/artipie/artipie/issues/711#issuecomment-718730426

Please set the date in release notes when release is ready and we will be good to publish

**Comment by olegmoz** [comment]
@olenagerasimova `0.12` has been release, here is link to Docker Hub: https://hub.docker.com/layers/artipie/artipie/0.12/images/sha256-427d961b2c1c9cb9228c8e267b4a6451b3a5d66583cc479ebfb9e5d6f83ddf13?context=explore

**Comment by olenagerasimova** [comment]
@olegmoz thanks, added link

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, publishing

**Pull request merged**